lib/src/messages/responses/cache/data/list/list_fetch.dart

```diff
@@ -31,11 +31,16 @@ class ListFetchError extends ErrorResponseBase implements ListFetchResponse {
 }
 
 /// Indicates that the requested list was successfully retrieved from the cache and can be accessed by the fields `values` or `binaryValues`.
-class ListFetchHit implements ListFetchResponse {
+class ListFetchHit extends NonErrorResponseBase implements ListFetchResponse {
   ListFetchHit(this._values);
 
   final List<List<int>> _values;
 
   Iterable<String> get values => _values.map((e) => utf8.decode(e));
   Iterable<List<int>> get binaryValues => _values;
+
+  @override
+  String toString() {
+    return "$runtimeType: $values";
+  }
```

Comment:
Response classes like this one (and including plain scalar `get`) might contain really really long values, so we've generally added a utility method in other SDKs to truncate values that may be obnoxious to display. The js sdk has a [set of utility functions](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/core/src/internal/utils/display.ts) that are called from appropriate `toString` methods you can look at for a quick example.